src/pages/groupComparison/vennDiagram/VennSimple.tsx

```diff
@@ -118,7 +91,7 @@ export default class VennSimple extends React.Component<IVennSimpleProps, {}> {
                 break;
         }
 
-        return computeVennJsSizes(combinations.map(combination=>{
+        const regions = combinations.map(combination=>{
             // compute the cases in this region
             let casesInRegion = _.intersection(...combination.map(index=>this.props.groups[index].cases));
```

Comment:
improve naming and better encapusulate calculation of casesInRegion to improve legibility